scripts/troubleshoot/LogCollection/README.md

```diff
@@ -1,19 +1,20 @@
-# Container Insights Log collector
+# Container Insights Log Collector
 
 This tool will collect:
-* agent logs from linux ds and rs pods;
-* agent logs from windows pod if enabled;
-* cluster/node info, pod deployment, configMap, process logs etc..
+* Agent logs from Linux ds (ama-logs-) and rs (ama-logs-rs-) pods
+* Agent logs from Windows pod if enabled
+* Cluster/node info, pod deployment, configMap, process logs etc.
+* Note: Script can collect logs from both AKS Clusters as well as ARO Clusters
 
 ## Prerequisites
 * kubectl: az aks install-cli
 * tar (installed by default)
-* all nodes should be running on AKS
```

Comment:
This pre-requisite shouldn't deleted. May be we can mention, nodes should be on AKS or running ARO?

Reply:
Agreed, I went ahead and added that to the prereqs.